## 52_gnus-pers.diff by Elias Oltmanns <oltmanns@uni-bonn.de>

--- a/elisp/gnus-bonus-el/gnus-pers.el	2004-08-16 20:49:16.000000000 -0400
+++ b/elisp/gnus-bonus-el/gnus-pers.el	2004-08-16 20:51:59.000000000 -0400
@@ -868,7 +868,8 @@
 	     (retval (nthcdr 2 split))
 	     partial regexp)
 	;; Check to see if it's a "gnus-newsgroup-name" split
-	(if (equal ",gnus-newsgroup-name" (cdr (assq field gnus-personality-split-abbrev-alist)))
+	(if (and gnus-newsgroup-name
+		 (equal ",gnus-newsgroup-name" (cdr (assq field gnus-personality-split-abbrev-alist))))
 	    (let ((groupname gnus-newsgroup-name))
 	      (if (string-match value groupname)
 		  retval))
